test/malli/swagger_test.cljc

```diff
@@ -276,3 +276,142 @@
                                                        [:zip int?]
                                                        [:country "Country"]]]]]]}}
            "Order"]))))
+
+(deftest swagger-spec-test
+  (testing "generates swagger for ::parameters w/ basic schema"
+    (let [registry (merge (m/type-schemas)
+                          {::body [:string {:min 1}]})]
+      (is (= {:definitions {::body {:minLength 1, :type "string"}},
+              :parameters [{:description "",
+                            :in "body",
+                            :name "body",
+                            :required true,
+                            :schema {:$ref "#/definitions/malli.swagger-test~1body",
+                                     :definitions {::body {:minLength 1, :type "string"}}}}]}
+             (swagger/swagger-spec {::swagger/parameters
+                                    {:body (m/schema ::body
+                                                     {:registry registry})}})))))
```

Comment:
Thanks for these! Could you also have a simple non-registry test case? Something like
```
{:body [:map [:foo :string]]}
```
?

Reply:
Added in [2fedffc](https://github.com/metosin/malli/pull/863/commits/2fedffc14e30c7ee90048ed31a472778089c3524)